CONCEPTS.md

```diff
@@ -54,10 +54,11 @@ clients and servers. A concrete embedding over HTTP/2 completes the picture by
 fleshing out the details of each of the required operations.
 
 ## Abstract gRPC protocol
-A gRPC call comprises of a bidirectional stream of messages, initiated by the client. In the client-to-server direction, this stream begins with a mandatory `Call Header`, followed by optional `Initial-Metadata`, followed by zero or more `Payload Messages`. The server-to-client direction contains an optional `Initial-Metadata`, followed by zero or more `Payload Messages` terminated with a mandatory `Status` and optional `Status-Metadata` (a.k.a.,`Trailing-Metadata`).
+A gRPC call comprises of a bidirectional stream of messages, initiated by a client. In the client-to-server direction, this stream begins with a mandatory `Call Header`, followed by optional `Initial-Metadata`, followed by zero or more `Payload Messages`. A client signals end of his message stream by means of an underlying lower level protocol. The server-to-client direction contains an optional `Initial-Metadata`, followed by zero or more `Payload Messages` terminated with a mandatory `Status` and optional `Status-Metadata` (a.k.a.,`Trailing-Metadata`).
```

Comment:
I think the phrase "underlying lower level protocol" is a bit vague here.  I'm not sure that including this sentence actually makes this document easier to understand.

Reply:
in HTTP/2 case, end of client's stream is signaled by closing the given HTTP/2 stream basically: no gRPC specific "frame" or "message" is used. As this section abstracts from HTTP/2 specifics, I thought that's the best way to describe this...
Any suggestions how to improve this maybe?